External/FEXCore/Source/Common/SoftFloat.h

```diff
@@ -0,0 +1,253 @@
+#pragma once
+#include "LogManager.h"
+
+#include <cmath>
+#include <cstring>
+#include <stdint.h>
+#include <string>
+#include <sstream>
+
+extern "C" {
+#include "SoftFloat-3e/platform.h"
+#include "SoftFloat-3e/softfloat.h"
+}
+
+struct X80SoftFloat {
+#ifdef _M_X86_64
```

Comment:
Maybe we should use some more descriptive and less magic defines here? It's kinda hard to keep track of all those esoteric names imo